compiler/qsc_fir/src/fir.rs

```diff
@@ -0,0 +1,1423 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+//! The high-level intermediate representation for Q#. HIR is lowered from the AST.
```

Comment:
Is this comment about HIR or FIR? (And what does 'FIR' stand for?)

Reply:
Flattened IR, I updated the comment.